Zukini.UI/CoypuExtensions.cs

```diff
@@ -1,8 +1,6 @@
 ﻿using Coypu;
-using OpenQA.Selenium;
 using System;
 using System.Collections.Generic;
-using System.Drawing;
 using System.Linq;
 
 namespace Zukini.UI
```

Comment:
Is this file now dead then? Should it be removed from the project?

Reply:
no, CoypuExtensions still contains the table row and cells extensions where as ElementScopeExtension deals specifically with ElementScope and the navtive WebElement.